src/views/Transaction/SendMultisigTx.tsx

```diff
@@ -0,0 +1,405 @@
+            } else {
+              throw Error(
+                'Failed to parse json file. Please check keystore file again.'
+              )
+            }
+          }
+        } catch (err) {
+          setDecryptMsg({
+            success: false,
+            msg: _.toString(err),
+          })
+        }
+      }
+    }
+  }
+
+  const decryptSenderKeystore = (): void => {
+    try {
+      if (!!senderKeystoreJSON) {
+        const keyring = caver.wallet.keyring.decrypt(
+          senderKeystoreJSON,
+          senderKeystorePassword
+        )
+        const privKeyList: string[] = []
+        if (keyring.type === 'SingleKeyring') {
+          privKeyList.push((keyring as SingleKeyring).key.privateKey)
+        } else if (keyring.type === 'MultipleKeyring') {
+          for (const element of (keyring as MultipleKeyring).keys) {
+            privKeyList.push(element.privateKey)
+          }
+        } else if (keyring.type === 'RoleBasedKeyring') {
+          const txRoleKeys = keyring.getKeyByRole(
+            caver.wallet.keyring.role.roleTransactionKey
+          )
+          for (const element of txRoleKeys) {
+            privKeyList.push(element.privateKey)
+          }
+        }
+        setDecryptMsg({
+          msg: 'Decryption succeeds!',
+          success: true,
+        })
+        setKeyList([
+          ...keyList,
+          { privateKeys: privKeyList, filename: keystoreFileName },
+        ])
+      } else {
+        throw Error('Keystore is not uploaded!')
+      }
+    } catch (err) {
+      setDecryptMsg({ success: false, msg: _.toString(err) })
+    }
+  }
+
+  const handleKeystoreRemove = (index: number): void => {
+    const privKeyList = [...keyList]
+    privKeyList.splice(index, 1)
+    setKeyList(privKeyList)
+  }
+
+  const onSignTxButtonClick = async (): Promise<void> => {
+    try {
+      setButtonDisabled(true)
+      let storedKeys: string[] = []
+      for (const element of keyList) {
+        storedKeys.push(...element.privateKeys)
+      }
+      const newKeyring = caver.wallet.keyring.createWithMultipleKey(
+        senderAddress,
+        storedKeys
+      )
+      if (caver.wallet.isExisted(senderAddress)) {
+        caver.wallet.updateKeyring(newKeyring)
+      } else {
+        caver.wallet.add(newKeyring) // caver wallet add keyring if keyring hasn't been updated.
+      }
+
+      let signed
+      if (tokenType === TokenTypeEnum.KLAY) {
+        //KLAY
+        const vt = caver.transaction.valueTransfer.create({
+          from: senderAddress,
+          to: recipientAddress,
+          value: caver.utils.toPeb(amount, 'KLAY'),
+          gas: 1000000,
+        })
+        signed = await caver.wallet.sign(senderAddress, vt)
+      } else {
+        //KIP-7 & ERC20
+        const contractInstance = new caver.kct.kip7(contractAddress)
+        const decimal = await contractInstance.decimals()
+        const value = BigNumber(_.toNumber(amount) * Math.pow(10, decimal))
```

Comment:
Bignumber 변환은 UTIL.toBn 만든거 있어요. 
```suggestion
        const value =  UTIL.toBn(amount).multipliedBy(Math.pow(10, decimal))
```